tools/distrib/python/grpc_prefixed/templates/README.rst.template

```diff
@@ -0,0 +1 @@
+The official package of {{ name_long }} is `{{ destination_package }} <https://pypi.org/project/{{ destination_package }}/>`_.
```

Comment:
Can we add "Please download that package instead." to make things a bit more explicit?

Reply:
Line added 👍